test/foundry/marketRegistry/MarketRegistry.t.sol

```diff
@@ -0,0 +1,211 @@
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import "../helper/Helper.sol";
+import "../helper/Setup.sol";
+import "../../../contracts/ClearingHouse.sol";
+import "../interface/IMarketRegistryEvent.sol";
+import "@uniswap/v3-core/contracts/interfaces/IUniswapV3Factory.sol";
+import "@uniswap/v3-core/contracts/interfaces/pool/IUniswapV3PoolState.sol";
+import "@uniswap/v3-core/contracts/UniswapV3Pool.sol";
+
+contract MarketRegistryAddPoolTest is IMarketRegistryEvent, Helper, Setup {
+    function setUp() public virtual override {
+        Setup.setUp();
+    }
+
+    function test_addPool_should_emit_event() public {
+        // add first pool
+        vm.mockCall(
+            address(pool),
+            abi.encodeWithSelector(IUniswapV3PoolState.slot0.selector),
+            abi.encode(100, 0, 0, 0, 0, 0, false)
+        );
+
+        _expect_event_emitted_from(address(marketRegistry));
+        emit PoolAdded(address(baseToken), _DEFAULT_POOL_FEE, address(pool));
+        marketRegistry.addPool(address(baseToken), _DEFAULT_POOL_FEE);
```

Comment:
if we use `baseToken2` here, the test will pass as well, but we expect it will fail